crates/ra_hir_def/src/adt.rs

```diff
@@ -57,18 +60,51 @@ pub struct FieldData {
     pub visibility: RawVisibility,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum ReprKind {
+    Packed,
+    Other(TokenTree),
```

Comment:
not sure if we should store the TokenTree in this case

Reply:
Removed